locations/hours.py

```diff
@@ -378,6 +378,34 @@
     "Duminică": "Su",
 }
 
+DAYS_SR = {
```

Comment:
Looks like this is Croatian? If so, can you make this `DAYS_HR`?

Reply:
I think it's correct as `SR` because locations are in Serbia where the dominant language is ISO-639-1 language code `SR`, not Croatia with the dominant language of ISO-639-1 language code `HR`?

Reply:
Yea, makes sense. I was trusting Google Translate too much.